experiment/example/example_test.go

```diff
@@ -0,0 +1,46 @@
+package example_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/apex/log"
+	"github.com/ooni/probe-engine/experiment/example"
+	"github.com/ooni/probe-engine/session"
+)
+
+const (
+	softwareName    = "ooniprobe-example"
+	softwareVersion = "0.0.1"
+)
+
+func TestIntegration(t *testing.T) {
+	log.SetLevel(log.DebugLevel)
+	ctx := context.Background()
+
+	sess := session.New(
+		log.Log, softwareName, softwareVersion, "../../testdata", nil, nil,
+	)
+	if err := sess.MaybeLookupBackends(ctx); err != nil {
+		t.Fatal(err)
+	}
+	if err := sess.MaybeLookupLocation(ctx); err != nil {
+		t.Fatal(err)
+	}
+
+	experiment := example.NewExperiment(
+		sess, example.Config{SleepTime: 10 * time.Second})
```

Comment:
In the next changes I will take parenthesis to the next line.
(In this case it will not be out of structure)